### PR TITLE
Signup using facebook 

### PR DESCRIPTION
**Pull request by Israa-Elwan**
@74 , User can sign up using his facebook account

### CODE REVIEW

**Comment by Israa-Elwan** [comment]
@Veegas  code review
@Mernawaseif UML review
@MinaNemr documentation review

**Comment by Mernawaseif** [comment]
@Israa-Elwan UMLs reviewed.  Nothing to change.

**Comment by Israa-Elwan** [comment]
@nelsammak scenario review

File: backend/config/passport.js [hunk 1]
```diff
@@ -39,22 +39,85 @@ module.exports = function () {
...
+        clientID        : configAuth.facebookAuth.clientID,
+        clientSecret    : configAuth.facebookAuth.clientSecret,
+        callbackURL     : configAuth.facebookAuth.callbackURL
+
+    },
+
+    // facebook will send back the token and profile
+    function(token, refreshToken, profile, done) {
+
+        // asynchronous
```
Comment (MinaNemr) [inline backend/config/passport.js:82]: @Israa-Elwan 
you should name this function and document using a form like this
/**
- @function getUsers - Called on GET "/api/users" 
- @params {Object} req - Http request
- @params {Object} res - Http response
- @params {Object} next - Next middleware
- @returns {JSON} All users in Database as JSON
  */

File: frontend/views/partials/landing.html [hunk 2]
```diff
@@ -37,9 +55,16 @@
             <button class="btn btn-lg btn-primary" type="button" ng-click="loginGo()">Login</button>
             &nbsp;
              <button class="btn btn-lg btn-primary"  type="button" ng-click="signupGo()">Sign Up</button>
```
Comment (MinaNemr) [inline frontend/views/partials/landing.html:61]: @Israa-Elwan 
please add section identifier to know what begins where
like
<!-edit form starts here-->

Html code
<!-edit form ends here-->

**Pull request merged**